### ISSUE TITLE
[Rule Request] Computed Property Accessors Order

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#3158]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### New rule request

Similar to [Protocol Property Accessors Order](https://realm.github.io/SwiftLint/protocol_property_accessors_order.html), but this rule should validate computed properties bodies so a certain order is enforced.

1. Why should this rule be added? Share links to existing discussion about what
   the community thinks about this.

We already have https://realm.github.io/SwiftLint/protocol_property_accessors_order.html

2. Provide several examples of what _would_ and _wouldn't_ trigger violations.

```swift
// ok
var foo: String?
var foo: String { return "" }
var foo: String {
  get { "" }
  set { print(newValue) }
}

// not ok
var foo: String {
  set { print(newValue) }
  get { "" }
}
```

3. Should the rule be configurable, if so what parameters should be configurable?

- Severity
- Order (`get set` x `set get`)

4. Should the rule be opt-in or enabled by default? Why?
   See [README.md](../README.md#opt-in-rules) for guidelines on when to mark a rule as opt-in.

This should be enabled by default to follow https://realm.github.io/SwiftLint/protocol_property_accessors_order.html.
